### ISSUE TITLE
Fill portal with particles

### ISSUE THREAD

**Issue by ZeaLBG** [Multiverse/Multiverse-Portals#322]
Add command to fill the portal whit particles (like when you place water)
Command: `/mvpm fill {water ; particle name}`
Example: `/mvpm fill enchantmenttable`
Particles list: http://minecraft.gamepedia.com/Particles#section_1

**Comment by farum12** [comment]
Hello, 
Are there any plans regarding that feature? That would increase immersion/user experience of portals created with MV.
Thanks!

**Comment by benwoo1110** [comment]
no, not at the moment. But it will be left open for anyone interested in opening a good PR to add this feature.

**Comment by jippi** [comment]
I might be interested in sponsoring this effort financially if someone can give me a rough estimate :)

**Comment by tkiehl** [comment]
Portal effects, please. :-)
